samples/matter/light_switch/prj_no_dfu.conf

```diff
@@ -31,3 +31,6 @@ CONFIG_CHIP_QSPI_NOR=n
 
 # Reduce application size
 CONFIG_USE_SEGGER_RTT=n
+
+# Enable the Read Client for binding purposes
```

Comment:
Minor: I got your mental shortcut, but read client as name suggests is used for reading attributes, not binding purposes :smile: